From 48f15935cff53836eae4c1c9afb0be8ab4f13bea Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Mon, 16 Jan 2023 15:12:16 +0000
Subject: [PATCH] convert: windows: Remove unused 'open Utils'

No function from Utils was being used by Windows_virtio, so remove the
unnecessary line.

(cherry picked from commit 26b9cd1f87f06c31cdf3020e003078737a417889)
---
 convert/windows_virtio.ml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/convert/windows_virtio.ml b/convert/windows_virtio.ml
index 2ce92ece..6b642d21 100644
--- a/convert/windows_virtio.ml
+++ b/convert/windows_virtio.ml
@@ -25,7 +25,6 @@ open Common_gettext.Gettext
 open Regedit
 
 open Types
-open Utils
 
 module G = Guestfs
 
